integration-tests-en/interactions/bot/test_navigation/explanations.jinja

```diff
@@ -0,0 +1,35 @@
+[
+    {
+        "text": "I can help you find testing sites near you"
+    },
+    {
+        "text": "Depending on the area where you live, or the testing site, you may need to have a referral from your doctor or a nurse in order to get tested"
+    },
+    {
+        "text": "You can get a referral by either: \n- Talking to your doctor, if that’s possible, or \n- Calling 811 and discussing your symptoms with a nurse"
```

Comment:
Should 811 be dynamic?

Reply:
Humm. I didn't think about it because we don't have the province in this situation, but yes, it should be, and default to this